statsmodels/base/transform.py

```diff
@@ -0,0 +1,222 @@
+import numpy as np
+from statsmodels.robust import mad
+from scipy.optimize import minimize_scalar
+
+
+class BoxCox(object):
+    """
+    Mixin class to allow for a Box-Cox transformation.
+    """
+
+    def transform_boxcox(self, x, lmbda=None, method='guerrero', **kwargs):
+        """
+        Performs a Box-Cox transformation on the data array x. If lmbda is None,
```

Comment:
I am reasonably sure we should not want to diverge too much from the literature. In any case, SciPy also uses `lmbda` for their implementation of the BoxCox transform (see https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.boxcox.html).